code/edgebridge/build.gradle

```diff
@@ -213,8 +213,7 @@ dependencies {
 
     testImplementation "androidx.test.ext:junit:${rootProject.ext.junitVersion}"
     testImplementation "org.mockito:mockito-core:${rootProject.ext.mockitoCoreVersion}"
-    testImplementation 'org.powermock:powermock-api-mockito2:2.0.0'
-    testImplementation 'org.powermock:powermock-module-junit4:2.0.0'
+    testImplementation 'org.mockito:mockito-inline:4.5.1'
```

Comment:
do these libs evolve together, can we use rootProject.ext.mockitoVersion for both core and inline?